core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -194,4 +226,43 @@ public String getServiceHost(String serviceName, Integer servicePort) {
     public Integer getServicePort(String serviceName, Integer servicePort) {
         return ambassadorContainers.get(serviceName + ":" + servicePort).getMappedPort(servicePort);
     }
+
+    public SELF withScaledService(String serviceBaseName, int numInstances) {
+        scalingPreferences.put(serviceBaseName, numInstances);
+
+        return (SELF) this;
+    }
 }
+
+class DockerCompose extends GenericContainer<DockerCompose> {
```

Comment:
This is the actual container implementation. Arguably this should now be named `DockerComposeContainer` and the `@Rule` implementation should not; I've done it this way for backward compatibility...